chunky/src/java/se/llbit/math/Octree.java

```diff
@@ -37,8 +37,8 @@
 import se.llbit.chunky.renderer.scene.Scene;
 import se.llbit.chunky.world.Material;
 import se.llbit.log.Log;
-import se.llbit.util.PositionalInputStream;
-import se.llbit.util.PositionalOutputStream;
+import se.llbit.util.io.PositionalInputStream;
```

Comment:
Is moving these classes really necessary?

Reply:
I like structure, especially in the util-package :)
Not really necessary, but there are many classes in it which do very different things. And I did not expect plugins to use these IO-classes for now.